oauth-jwt/jwt-auth-server/src/main/resources/baeldung-realm.json

```diff
@@ -936,6 +936,14 @@
       "display.on.consent.screen" : "true"
     }
   }, {
```

Comment:
Just for my information: Why is this needed?

Reply:
This is a client setting - If enabled, users have to consent to client access.